src/resources/postcss/tickets-refresh.pcss

```diff
@@ -105,6 +126,18 @@
 	margin-bottom: 20px;
 }
 
+.tribe_no_capacity {
+	margin: 0;
+}
+
+.ticket_delete {
+	color: #a00;
+
+	&:hover {
+		color: #dc3232;
```

Comment:
if this and above color for `.ticket_delete` are used more than once, they may be a variable ... are they? if so, replace these values with the variable. if their is no variable for these values, ignore this stupid comment.